app/services/payment_providers/stripe/payments/create_service.rb

```diff
@@ -78,8 +78,9 @@ def stripe_payment_method
           end
 
           # NOTE: Retrieve list of existing payment_methods
-          payment_method = ::Stripe::PaymentMethod.list(
-            {customer: provider_customer.provider_customer_id},
+          payment_method = ::Stripe::Customer.list_payment_methods(
+            provider_customer.provider_customer_id,
```

Comment:
Noticed that I changed this call because it's easier to mock the http request. The response payload is the same.